ServerRoomMonitoring.Generator/Messaging/ServerRoomMq.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text;
+using RabbitMQ.Client;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace ServerRoomMonitoringGenerator.Messaging
+{
+    public class ServerRoomMq
```

Comment:
Ta klasa powinna sama tworzyć połączenie na początku w konstruktorze (albo z pomocą fabryki połączeń, żeby było jeszcze bardziej czysto w kodzie, która by zwróciła od razu kolejkę o podanej nazwie). 
Połączenie może funkcjonować w ramach jednego wątku, a ta zależność bardziej tutaj pasuje.
Fajnie też by było tak ustawić konfigurację apki, żeby wspierała DependencyInjection. Możemy się jutro spotkać to pokażę jak to można ładnie sensownie zrobić.

Reply:
I wtedy sensor miałby metodę generującą taki obiekt DTO, a nie musiałby mieć pól temperatura zdefiniowanych jako pola. Generator powinien mieć jako pola tylko swoją konfigurację typu zakres temperatur, częstotliwość odczytów itp. zależnie od potrzeb w późniejszych etapach.